ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -208,6 +208,9 @@ def create_policy(
 
         return policy
 
+    def create_ppo_optimizer(self) -> PPOOptimizer:
```

Comment:
Made these so we can wrap the TFOptimizer impls more easily. I briefly tried adding
```
    @abc.abstractmethod
    def create_tf_optimizer(self) -> TFOptimizer:
```
in RLTrainer but I think at least SACTrainer needs the specific type. Do you agree, or should I try harder to push this higher in the hierarchy?

Reply:
LGTM